api/apps/api/src/modules/scenarios/marxan-run/assets.service.ts

```diff
@@ -12,33 +16,33 @@ export const apiUrlProvider: FactoryProvider<string> = {
 @Injectable()
 export class AssetsService {
   constructor(
-    @Inject(ioSettingsToken)
-    private readonly settings: IoSettings,
     @Inject(apiUrlToken)
     private readonly apiUrlToken: string,
+    private readonly inputFiles: InputFilesService,
   ) {}
 
   async forScenario(id: string): Promise<Assets> {
+    const settings = await this.inputFiles.getSettings();
```

Comment:
theoretically could be left untouched with a proper factory provider

but ok :)